neurodamus/core/_engine.py

```diff
@@ -77,23 +77,3 @@ class EngineBase(metaclass=_EngineMeta):
     """A dict of the new connection types and associated Manager class"""
     CircuitPrecedence = 1
     """Precedence influences instantiation order. The lower the earlier setup"""
-
-    @classmethod
-    def new_cell_manager(cls, circuit_conf, target_parser, run_conf):
-        if not cls.CellManagerCls:
-            raise RuntimeError("missing CellManagerCls")
-        return cls.CellManagerCls(circuit_conf, target_parser, run_conf)
-
-    def new_synapse_manager(cls, circuit_conf, target_manager, cell_manager):
-        """Routine responsible for creating connection managers."""
-        if not cls.SynapseManagerCls:
```

Comment:
Can you also update the docstring of class `EngineBase` ?  `SynapseManagerCls` -> `InnerConnectivityCls`

Reply:
sure, will do